app.js

```diff
@@ -1,53 +1,87 @@
-const FileTextEditor = require("./textEditorService");
-const parsers = require("./parsers");
+const fs = require("fs/promises");
+const { bootstrap } = require("kaholo-plugin-library");
+const { tryCreateRegexFromString, parsePath, pathExists } = require("./helpers");
 
-function createFile(action) {
-  const fileEditor = new FileTextEditor(action.params.PATH);
-  return fileEditor.createFile({
-    content: action.params.CONTENT,
-  });
+async function createFile({
+  PATH: path,
+  CONTENT: content,
+}) {
+  const parsedPath = parsePath(path);
+  try {
+    await fs.writeFile(parsedPath, content);
+  } catch (error) {
+    throw new Error(`Error creating file ${path}: ${error.message || JSON.stringify(error)}`);
```

Comment:
In such situation the output will result in double or even tripple `Error` keyword repetition. Please state what could be wrong here without using that word, which is appended to the output at the platform level anyway.